analysed-packages/ppp/version-2.4.9/ppp-2.4.9-SPDX2TV.spdx

```diff
@@ -0,0 +1,5119 @@
+SPDXVersion: SPDX-2.3
+DataLicense: CC0-1.0
+
+##-------------------------
+## Document Information
+##-------------------------
+
+DocumentNamespace: http://fossy.osadl.org/repo/SPDX2TV_ppp-2.4.9.tar.gz_1761842334.spdx
+DocumentName: /srv/fossology/repository/report
+SPDXID: SPDXRef-DOCUMENT
+
+##-------------------------
+## Creation Information
+##-------------------------
+
+Creator: Tool: fossology-4.4.0.51
+Creator: Person: Astrid Spura (aspura@osadl.org)
+CreatorComment: <text>This document was created using license information and a generator from Fossology. It contains the license and copyright analysis of ppp 2.4.9.
+Please check "LicenseComments" for explanations of concluded licenses.
+</text>
+Created: 2025-10-30T16:38:55Z
+LicenseListVersion: 3.22
+
+##-------------------------
+## Package Information
+##-------------------------
+
+
+PackageName: ppp-2.4.9.tar.gz
+PackageFileName: ppp-2.4.9.tar.gz
+SPDXID: SPDXRef-upload3512
+PackageDownloadLocation: https://download.samba.org/pub/ppp/ppp-2.4.9.tar.gz
+ExternalRef: PACKAGE-MANAGER purl pkg:generic/ppp@2.4.9?download_url=https://download.samba.org/pub/ppp/ppp-2.4.9.tar.gz
+FilesAnalyzed: true
+PackageVerificationCode: 5064e2289958718723bdf56808f357c38915cee5
+PackageChecksum: SHA1: 56ba143f331f6436efa3a1d41c75691c828833ba
+PackageChecksum: SHA256: f938b35eccde533ea800b15a7445b2f1137da7f88e32a16898d02dee8adc058d
+PackageChecksum: MD5: c88153ae3d16ae114152cd3c15c7301d
+PackageLicenseConcluded: NOASSERTION
+PackageLicenseDeclared: LicenseRef-PPP-Copyright
+PackageLicenseComments: <text> licenseInfoInFile determined by Scanners:
+ - nomos ("4.4.0.51".1e52cb)
+ - monk ("4.4.0.51".1e52cb)
+ - ojo ("4.4.0.51".1e52cb)
+ - scancode ("4.4.0.51".1e52cb) </text>
+PackageLicenseInfoFromFiles: NOASSERTION
+PackageCopyrightText: NOASSERTION
+
+Relationship: SPDXRef-DOCUMENT DESCRIBES SPDXRef-upload3512
+
+
+##--------------------------
+## File Information
+##--------------------------
+
+##File
+
+FileName: ppp-2.4.9.tar.gz/ppp-2.4.9.tar/ppp-2.4.9/README
+SPDXID: SPDXRef-item16155440
+FileChecksum: SHA1: 1c47206c94a4c8eb728de349666a5b2332f79c55
+FileChecksum: SHA256: c5591c60b8cd1f1250dbea691fbb1ed162b754134a8cfd462930626e5f13a72b
+FileChecksum: MD5: 6f64175fd49641c4ca231238b341eeef
+LicenseConcluded: LicenseRef-PPP-Copyright
```

Comment:
This is not a license. Further I do not see that this file is "licensed" under the "license". in the file it is stated:
All of the code can be freely used and redistributed. The individual
source files each have their own copyright and permission notice.
Pppd, pppstats and pppdump are under BSD-style notices. Some of the
pppd plugins are GPL'd. Chat is public domain.
This file contains no code and is not a file which is mentioned.